src/TwitchLib.Communication/TwitchLib.Communication.csproj

```diff
@@ -1,24 +1,32 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <VersionPrefix>1.0.6</VersionPrefix>
-    <VersionSuffix>$(VersionSuffix)</VersionSuffix>
-    <Authors>swiftyspiffy, Prom3theu5, Syzuna, LuckyNoS7evin</Authors>
-    <Company>swiftyspiffy, Prom3theu5, Syzuna, LuckyNoS7evin</Company>
-    <Description>Connection library used throughout TwitchLib to replace third party depedencies.</Description>
-    <Copyright>Copyright 2022</Copyright>
-    <PackageLicenseUrl>https://opensource.org/licenses/MIT</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/TwitchLib/TwitchLib.Communication</PackageProjectUrl>
-    <PackageIconUrl>https://cdn.syzuna-programs.de/images/twitchlib.png</PackageIconUrl>
-    <RepositoryUrl>https://github.com/TwitchLib/TwitchLib.Communication</RepositoryUrl>
-    <RepositoryType>Git</RepositoryType>
-    <PackageTags>twitch twitchlib library irc chat c# csharp api events pubsub net standard 2.0</PackageTags>
-    <PackageReleaseNotes>Fix reconnect loop on disconnect</PackageReleaseNotes>
-    <NeutralLanguage>en-US</NeutralLanguage>
-    <AssemblyVersion>1.0.6</AssemblyVersion>
-    <FileVersion>1.0.6</FileVersion>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-  </PropertyGroup>
-
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net7.0</TargetFrameworks>
+        <Nullable>enable</Nullable>
+        <LangVersion>latest</LangVersion>
+        <VersionPrefix>2.0.0</VersionPrefix>
+        <VersionSuffix>$(VersionSuffix)</VersionSuffix>
+        <Authors>swiftyspiffy, Prom3theu5, Syzuna, LuckyNoS7evin</Authors>
+        <Company>swiftyspiffy, Prom3theu5, Syzuna, LuckyNoS7evin</Company>
```

Comment:
feel like we should add some reference to Github contributors, since a decent portion of this code is now written by new-ish people :)

Reply:
I am fine with that.
How would you like to do that?

Reply:
I'm not sure it's realistic to include every new contributor in this list, so maybe just "swiftyspiffy, Prom3theu5, Syzuna, LuckyNoS7evin, and Github contributors" or something? We'd prob need to do something similar for the other packages as well.

Reply:
We can add this later though, lets not block on this.

Reply:
Yeah I would like to add the one other PR to this before merging tho.
Did you have time to look at that one too? Bcs I didnt yet

Reply:
Referring to #27 ? I can look tonight (in a few hours).